Prerequisites-notebook.ipynb

```diff
@@ -0,0 +1,246 @@
+{
+ "cells": [
+  {
```

Comment:
Is the user already has these pre, requisites can we take them as input or if they want to create it then create it for them

Reply:
done